### PR TITLE
fix documentation build errors

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by greg0ire** [comment]
@xabbuh should this maybe target a lower branch?

**Comment by greg0ire** [comment]
Looks like you already did in https://github.com/doctrine/DoctrineMigrationsBundle/pull/344, sorry, I didn't notice.

**Comment by SenseException** [comment]
Is an upmerge to master the prefered way or should this PR introduce the changes?

**Comment by goetas** [comment]
It will take some time, but #344 will be upmerged to 3.x and master (there is already a first PR https://github.com/doctrine/DoctrineMigrationsBundle/pull/346 for that). In this way we avoid conflicts and duplicate commits.

Thanks @xabbuh !

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -97,6 +97,7 @@ application:
 
 - The ``services`` node allows you to provide custom services to the underlying ``DependencyFactory`` part of ``doctrine/migrations``.
 - The node ``factories`` is similar to ``services``, with the difference that it accepts only callables.
+
```
Comment (xabbuh) [inline Resources/doc/index.rst:100]: @goetas When merging up this blank line is important (it's not missing in the lower branch).

**Pull request merged**